packages/sync/src/Actions.php

```diff
@@ -263,24 +366,47 @@ static function do_full_sync( $modules = null ) {
 		return true;
 	}
 
-	static function jetpack_cron_schedule( $schedules ) {
+	/**
+	 * Adds a cron schedule for regular syncing via cron, unless the schedule already exists.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param array $schedules A list of WordPress cron schedules.
+	 * @return array
+	 */
+	public static function jetpack_cron_schedule( $schedules ) {
 		if ( ! isset( $schedules[ self::DEFAULT_SYNC_CRON_INTERVAL_NAME ] ) ) {
+			$minutes = intval( self::DEFAULT_SYNC_CRON_INTERVAL_VALUE / 60 );
+			$display = ( 1 === $minutes ) ?
+				__( 'Every minute', 'jetpack' ) :
+				/* translators: %d is an intereger indicating the number of minutes. */
+				sprintf( _n( 'Every %d minute', 'Every %d minutes', $minutes, 'jetpack' ), $minutes );
```

Comment:
Is this additional logic necessary because the `%d` flag must be used in both singular and plural?

Reply:
yes!